### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Divorso**
test

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # gruppovning
 
-hej grupp allihop hejehjehje
+hej grupp 1 testest
```
Comment (PerRosen63) [inline README.md:3]: Helt ok, bra jobbat, men test står dubbelt!!! Fixa!

**Review by PerRosen63 — COMMENTED** [review]
Uppfattat!

**Review by tsemitris — CHANGES_REQUESTED** [review]
Good job! In line 3 i see that you have wrote two times the word test can you remove it?

**Review by Divorso — COMMENTED** [review]
ok till ändringen

**Review by tsemitris — APPROVED** [review]
Ok

**Review by Divorso — COMMENTED** [review]
ok

**Review by PerRosen63 — COMMENTED** [review]
Ta bort ett test

**Review by PerRosen63 — APPROVED** [review]
Ok, kör!

**Review by tsemitris — CHANGES_REQUESTED** [review]
Good job! In line 3 i see that you have wrote two times the word test can you remove it?

**Comment by Divorso** [comment]
hej grupp allihop hejehjehje---> kommer ersättas av ------> hej grupp 1 testest

**Pull request merged**